src/components/search-box/SearchBox.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import './SearchBox.scss';
```

Comment:
this should be pulled in via dictionaries.scss

Reply:
Done